styles/main.css

```diff
@@ -49,308 +58,129 @@
 
     #id_sitecontainer{
         display: block;
-        font-family: 'Encode Sans Expanded', sans-serif;
-        pointer-events: none;
         z-index: 2;
     }
-
-        #id_aboutus{
```

Comment:
(Review comment 6) Let's debate this notion but I previously left empty CSS corresponding to each class and ID in each stylesheet, with the motivation being that if developer wanted to tweak a section the hooks were already in place to do it in an organized way - we can def take out any empty CSS references, but developer will have the extra step of finding the "right spot" in the CSS (unless we decide not to do use this organizational technique moving forward - would love to discuss multiple sides of the argument on this point and hear any fresh perspectives)